crescent/plugin.py

```diff
@@ -30,11 +30,6 @@ def __init__(self, bot: Bot) -> None:
         self.plugins: dict[str, Plugin] = {}
         self._bot = bot
 
-    def add_plugin(self, plugin: Plugin, force: bool = False) -> None:
-        _LOG.warning("`add_plugin` is deprecated and will be removed in a future release.")
-
-        self._add_plugin("", plugin, refresh=force)
-
```

Comment:
Probably already discussed this, but what's the alternative here?

Reply:
`PluginManager.load`

Reply:
Ah ok